README.md

```diff
@@ -41,7 +41,7 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
   - 1+ merged PR: Stickers
   - 3+ merged PR: T-Shirt
   - Top contributor Referral: WASD Keyboard.
-- **How to sign up**: Raise a PR and join us on discord.[Appsmith Hacktoberfest](https://github.com/appsmithorg/appsmith/discussions/863)
+- **How to sign up**: Raise a PR and join us on [discord](https://discord.gg/rBTTVJp). [Appsmith Hacktoberfest](https://github.com/appsmithorg/appsmith/discussions/863)
```

Comment:
If you wish to make this change, please also do it below in the Least Involvement to Most Involvement section, such as line number 373 below.